elixir/apps/web/lib/web/live/sites/edit.ex

```diff
@@ -29,12 +29,24 @@ defmodule Web.Sites.Edit do
             <div class="grid gap-4 mb-4 sm:grid-cols-1 sm:gap-6 sm:mb-6">
               <div>
                 <.input
-                  label="Name Prefix"
+                  label="Name"
                   field={@form[:name_prefix]}
                   placeholder="Name of this Site"
                   required
                 />
               </div>
+              <div>
+                <.input
+                  type="select"
+                  label="Routing"
+                  field={@form[:routing]}
+                  options={[
+                    [key: "Managed Relays", value: "managed"],
+                    [key: "No Relays (STUN only)", value: "stun_only"]
```

Comment:
I think we decided to show all the options but disable some of them with a note "Contact us to use it" or something like that. cc @jamilbk

Reply:
In this issue it explicitly says to only include the one's I've listed here:
https://github.com/firezone/firezone/issues/2472

Reply:
@jamilbk should we add "call to actions" to the dropdown? Anyway this LGTM

Reply:
Let's leave them out of the list until we're able to support it / have customer conviction around the other ones.